cmake-scripts/version.cmake

```diff
@@ -0,0 +1,225 @@
+# Calculate OMF_ROOT, the path to the root of the repo.
+if(DEFINED CMAKE_SCRIPT_MODE_FILE)
+    set(OMF_ROOT "${CMAKE_SCRIPT_MODE_FILE}")
+    get_filename_component(OMF_ROOT "${OMF_ROOT}" DIRECTORY)
+    get_filename_component(OMF_ROOT "${OMF_ROOT}" DIRECTORY)
```

Comment:
Duplicate line here, probably not deliberate?

Reply:
That is deliberate. One is to remove the `version.cmake` from the path, the second removes `cmake-scripts/` from the path.